.github/workflows/windows.yml

```diff
@@ -0,0 +1,49 @@
+name: Windows build and Package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
```

Comment:
add `fail-fast: false`

Reply:
Hm do you think it's just that specific versions can't be found? I guess worth a try.

Reply:
No, that wasn't it. It actually can't find MSBuild. See linked issues.

Reply:
It is not about MSBuild, it is about not canceling all matrix build if only one of the is failng.
Let others continue/finish independently.